test/Select-test.js

```diff
@@ -2389,6 +2389,69 @@ describe('Select', function() {
 			});
 		});
 
+		describe('searchingText', function () {
+
```

Comment:
because of the blank line after the `function () {`?

Eeee. Funnily enough, that's the [hapi](http://hapijs.org) style, which I like, and find it makes it much easier to read, I also saw that there is a blank line after many functions.

We'll need to do a clean up of all the tests if that gets accepted.  I really don't like that it seems to fail on node 0.10 though.